### PR TITLE
Add support for multicall tx

### PR DESCRIPTION
**Pull request by zainbacchus**
- Staging first changes to enable multicall

### CODE REVIEW

**Comment by annieke** [comment]
did you get to test this? don't see an sdk version change, so you might need to run something like `npm install @eth-optimism/sdk@canary` to use the canary sdk

**Comment by zainbacchus** [comment]
> did you get to test this? don't see an sdk version change, so you might need to run something like `npm install @eth-optimism/sdk@canary` to use the canary sdk

See https://github.com/ethereum-optimism/ecopod/issues/162 for current status of this issue! Not using the updated SDK yet.

File: src/components/Relayer.tsx [hunk 1]
```diff
@@ -99,6 +102,59 @@ export function Relayer() {
...
+    } else if (numMessages < 2 && messageStatus) {
+      return 'Message status: '  + messageStatus
+    } else if (numMessages > 1 && messageStatus) {
+      return 'Message status of message ' + indexValue + ': '  + messageStatus
+    } else {
+      return ''
+    }
+  }
+
+  const renderIndex = () => {
```
Comment (zainbacchus) [inline src/components/Relayer.tsx:130]: FYI @tarunkhasnavis

**Comment by zainbacchus** [comment]
> did you get to test this? don't see an sdk version change, so you might need to run something like `npm install @eth-optimism/sdk@canary` to use the canary sdk
> 
> Exact version is [this one](https://www.npmjs.com/package/@eth-optimism/sdk/v/0.0.0-20230711223728), which is also accessible with the [`@canary` tag](https://www.npmjs.com/package/@eth-optimism/sdk/v/0.0.0-2022812163333?activeTab=versions)

cc @tarunkhasnavis  re upgrading SDK if official release is not out by the time you pick this PR up

**Comment by zainbacchus** [comment]
Closing for #18

**Pull request closed without merging**